dto/src/main/java/org/jboss/pnc/dto/response/RunningBuildCount.java

```diff
@@ -20,12 +20,26 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
+/**
+ * Statistics of running builds.
+ *
+ * @author jbrazdil
+ */
 @AllArgsConstructor
 @Data
 public class RunningBuildCount {
 
+    /**
+     * Number of running builds.
+     */
     private int running;
+    /**
+     * Number of enqueued builds.
+     */
     private int enqueued;
+    /**
+     * Number of builds waiting for dependencies.
+     */
     private int awaitingForDependencies;
```

Comment:
Shouldn't it be called "waitingForDependencies" instead of "awaitingForDependencies"? On the other hand,
 changing name of an argument would break API on UI.

Reply:
@thescouser89 question for you.

Reply:
And the possible change should be handled in different issue.

Reply:
agreed

Reply:
+1 to rename it. We should do it for 2.0. Have you created the issue?